### ISSUE TITLE
OpenAI Approximate Cost for Demo?

### ISSUE THREAD

**Issue by weisisheng** [weaviate/healthsearch-demo#5]
Would be great to understand how much cost (range) would be incurred for setup and a few queries.

**Comment by corticalstack** [comment]
Some additional guidance and consideration around "setup from scratch" is needed, for example the openai api free tier is rate limited to 3 embeddings per minute, so the example `import_data_to_weaviate.py` fails with rate hard limits exceeded.

**Comment by thomashacker** [comment]
Thanks for the issue; this is a completely valid question! I added a section to the main `README` which goes into more detail on the costs. The demo mainly uses OpenAI models; for importing the dataset, we use the `ada-v2` model, and the import costs roughly $0.002. For the query and summary generation, we use the `GPT-4 `model because of its high-quality output and capabilities to understand user intentions; the costs are roughly $0.20/1k token ([a token is about four characters](https://help.openai.com/en/articles/4936856-what-are-tokens-and-how-to-count-them))

Here's a bit more info about the pricing: https://openai.com/pricing

For users that don't have access to `GPT-4`, I added instructions on how to swap to another OpenAI model, such as `gpt-3.5-turbo`. I agree that only using the OpenAI models makes this demo restrictive, and I'd love to provide alternatives to other LLM providers in the future. 

To solve the rate limitation issue for free tier users, I will provide the embeddings for the dataset soon.
